hw/ip/otp_ctrl/data/otp_ctrl_img_rma.hjson

```diff
@@ -43,6 +43,163 @@
                 {
                     name:  "OWNER_SW_CFG_DIGEST",
                     value: "0x0",
+                },
+                {
+                    name: "ROM_ALERT_CLASS_EN"
+                    // Set the enables to kAlertEnableNone.
+                    // See `alert_enable_t`
+                    // in sw/device/silicon_creator/lib/drivers/alert.h
+                    value: "0xa9a9a9a9",
```

Comment:
Does this mean that we need to configure OTP before enabling mask ROM execution in manufacturing? It would be nice if we didn't have to.

Reply:
That depends on which LC state enables ROM execution.  We should create configs which represent the manufacturing states and then add tests to make sure the mask ROM behaves appropriately in those states.

Reply:
Yes, currently `shutdown_init()` reads these values only if prod, prod_end, dev, and rma (alert handler is not configured in test_unlocked LC states).